lib/ex_dynamodb_local/worker.ex

```diff
@@ -0,0 +1,69 @@
+defmodule ExDynamodbLocal.Worker do
+  require Logger
+
+  import ExDynamodbLocal.Util
+
+  # defaults
+  @jar_filename Application.get_env(:ex_dynamodb_local, :jar_filename, "DynamoDBLocal.jar")
+  @lib_path Application.get_env(:ex_dynamodb_local, :lib_path, "DynamoDBLocal_lib")
+  @in_memory Application.get_env(:ex_dynamodb_local, :in_memory, false)
+  @port Application.get_env(:ex_dynamodb_local, :port)
```

Comment:
This shouldn't affect you here since this will only be run on test/dev, but in the future be careful setting module attributes from application env.

These values will be compiled in at build time which means if they're set via unix environment variables it won't pick up any changes.

The best way to handle these in the future is to just use functions instead of module attributes.

Reply:
Makes sense